how-to/use_terraform.rst

```diff
@@ -46,7 +46,7 @@ Juju Terraform provider
 To authenticate with JIMM the provider section in your Terraform plan needs to include 
 the ``client_id`` and ``client_secret`` generated by your identity provider. Please note that 
 you need to use a version of the `Juju Terraform provider <https://registry.terraform.io/providers/juju/juju/latest/docs>`_
-higher than ``0.12.0``.
+higher or equal to ``0.12.0``.
```

Comment:
Definitely not 0.12.0, because it doesn't work. I asked about the next version from the Juju Terraform Provider team and the next version will be `0.13.0`.